Libs/DICOM/Widgets/ctkDICOMPatientItemWidget.cpp

```diff
@@ -787,8 +787,7 @@ void ctkDICOMPatientItemWidget::onSeriesItemClicked()
     return;
     }
 
-  if (QApplication::keyboardModifiers() &&
-      (Qt::ControlModifier || Qt::ShiftModifier))
+  if (QApplication::keyboardModifiers() & (Qt::ControlModifier | Qt::ShiftModifier))
```

Comment:
If ``Ctrl+Shift are pressed, we do not select ... ?

cc: @Punzo

Reply:
having a look now, I will report asap

Reply:
@jcfr 

ok I have retested and it works as intended. That if makes it that you can do multiselections with Qt::ControlModifier | Qt::ShiftModifier from multiple study items (multiple tables)

thanks for fixing the warnings (: